### PR TITLE
Fix broken formatting in mkdocs

### PR DESCRIPTION
**Pull request by SBBCornell**
This branch fixes broken formatting in mkdocs. [Preview the changes here](https://docs.arxiv.org/labs/index.html). I have done a pass to all directories, all pages, including the “to do” items listed [here](https://docs.arxiv.org/index.html#todo-review-content-and-nav)... with the exception of this one:

_Are we trying to get TeX on this page on the subject heading "Notes on Note on La$\TeX$ accent commands" ? /help/prep.md_

I think it may be displaying as intended but would like @jimentwood or @jweiskoff to confirm.

**ALSO DONE:** 
- updated Labs content with Hugging Face, papers with code, Iarxiv, and other recent updates. 
- updated /brand directory.
- Moved CORR to new /help/cs directory.
- Improved many page titles (but not every single page).
- Deleted redirect pages.
- Redid custom styles in /stylesheets/extra.css. Removed inline and on-page style blocks where in use. 

**TO DO STILL:**
1) Per the changes above, the following apache redirects will need to be put in place:
- /about/brand_use > /brand/brand-guidelines.html
- /CORR/ > /help/cs/index.html
- /corr/advisorycommittee > /help/cs/index.html
- /corr/subjectclasses > /help/cs/index.html
- /corr/home > /help/cs/index.html

2) In addition, the general redirect rules that have been discussed are:
- info.arxiv.org > info.arxiv.org/about/index.html
- arxiv.org/help, arxiv.org/about, arxiv.org/new > info.arxiv.org/*

3) all internal links still use the .md extension. @bdc34 what is the best way to address that? A bulk find and replace? Or a redirect from .md to .html? Or something else?

4) Jim may do more reorganization of pages and directories this week, which would lengthen the list of individual redirects. he suggested we should track them in a table.

### CODE REVIEW

**Comment by bdc34** [comment]
> 3. all internal links still use the .md extension. @bdc34 what is the best way to address that? A bulk find and replace? Or a redirect from .md to .html? Or something else?

The links in the .md files should be to .md files and they get converted to .html.

File: source/brand/images.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Photographs can be overlayed with a [tagline](tagline.html) as shown here.
+![photo of CU Tech campus with tagline](images/brand-image-tagline.jpg){.mkd-full alt='An image of arXiv's home at Cornell Tech overlayed with the tagline arXiv Connects old and new ways of publishing.'}  
+
+##Illustrations
+[Go to the arXiv Illustration library](https://cornell.box.com/v/arXiv-illustration-library)
+
+Illustrations are used to represent abstract concepts or stakeholder groups. The illustrative style we use is informal and friendly. It is not slick or polished, but emphasizes thrift and mission focus, with a touch of gentle humor.
+
+<ul class="brand-toc">
+  <li><img src="images/brand-image-illustration-1.jpg" role="representation"></li>
```
Comment (jimentwood) [inline source/brand/images.md:23]: Do these need alt tags or are they OK in context?

**Comment by jimentwood** [comment]
I did a quick scan and added a couple minor questions.
Where there are any formatting differences in this new model we need to get them documented and then go over them together as a team.

**Comment by afromme** [comment]
Is it possible to get a couple of example screenshots of the new page layout? And/Or, could we set  up a meeting to go over it together? Thanks!

**Comment by jimentwood** [comment]
For screenshots - I was assuming at some point this would be deployed where we can work in branches that have continuous deployment of PRs and that would make it easier to work with and test links and formats.

**Comment by bdc34** [comment]
> For screenshots - I was assuming at some point this would be deployed where we can work in branches that have continuous deployment of PRs and that would make it easier to work with and test links and formats.

This is already being deployed vis CICD and you can see it here: https://docs.arxiv.org/

Usually it would be a preview but since the docs.arxiv.org site is not live (and we will be using info.arxiv.org) I thought we could use it.

**Comment by jimentwood** [comment]
It looks like some of the deployed pages are not current pages
This url https://docs.arxiv.org/help/stats/index.html shows the 2020 version of the page which does not match https://arxiv.org/help/stats

image

**Review by jimentwood — APPROVED** [review]

**Pull request merged**